app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,4 @@
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
discard these unnecessary changes you did for this file please

Reply:
After doing this change, I have to write these commands. Right?
git add .
git commit --amend -m "fix: hide tabs on the current relation empty state"
git push -f origin emptyState
Then when do I have to write this command ->
git commit --amend --author="Author Name <email@address.com>"

Reply:
I think you can do I in this order:
```
git add .
git commit --amend -m "fix: hide tabs on the current relation empty state"
git commit --amend --author="Author Name email@address.com"
git push -f origin emptyState
```
Don't forget to use your account email, so that GitHub recognizes you @mohak1283

Reply:
`git commit --amend --author="Author Name <email@address.com>"`

Reply:
Pay attention to the format, so you don't have to do this again @mohak1283

Reply:
Author name should be Mohak Gupta. Right? or my username?
and email address I have to write it in <> bracket or without them? @isabelcosta

Reply:
Right! use the brackets :)

Reply:
Okay will do this:+1: Thanks @isabelcosta

Reply:
![bugpic](https://user-images.githubusercontent.com/35039342/51412069-5eba0280-1b90-11e9-89e7-a60d433b35de.PNG)
This is coming after I wrote this command
git commit --amend --author="Author Name <email@address.com>"
@isabelcosta

Reply:
thats an in terminal editor, like vim or nano on Linux. do wwrite this `:wq`. Write that, not in the comments, in the terminal. (SHIFT + : ) + w + q
And then click ENTER, to submit that and leave the screen. w is for write and q is for quit @mohak1283

Reply:
Done the changes :+1: Thanks once again @isabelcosta :+1: